### ISSUE TITLE
What is the difference between  box_3d , 'box_8c', 'box_8co',box_4c,'box_4ca' encoding for a bounding box ??

### ISSUE THREAD

**Issue by vignesh628** [kujason/avod#189]

**Comment by kujason** [comment]
Please check the wiki page for more details on the data formats:
https://github.com/kujason/avod/wiki/Data-Formats

**Comment by vignesh628** [comment]
Hello @kujason . Thank You so much for reply. Could you please also explain the difference between 'box4c' and 'box4ca' as it was not mentioned in wiki page

**Comment by vignesh628** [comment]
Can you please also comment how the 'box4c' encoding is better than the 'box3d' encoding and how it helps in model ? Is it like reduces the number of parameters to learn by model ??

**Comment by vignesh628** [comment]
Hello @kujason ,
In the AVOD paper you have written, you have mentioned about Explicit Orientation Vector Regression. In this you have mentioned it uses orientation vector. Usage of regressed orientation vector resolves the ambiguity in bounding box estimation. Can you please elaborate this part in layman terms 
what is the orientation vector ? How we create it ? How it handles angle wrapping in -pi,pi  ? How usage of regressed orientation vector resolves ambiguity in bbox estimation ? what is the final layer shape for bounding box vector ? How we will extract the four possible orientations of bounding box and chooe closest to regressed orientation vector ?
As this orientation estimation is new thing comes in 3D object detection can you please comment on these would give clear idea for real understanding and usage of AVOD. THanks...
